polkadot-api/src/action.ts

```diff
@@ -0,0 +1,47 @@
+type WebViewActionPayload = Record<string, unknown>;
+
+export enum ACTION_TYPES {
```

Comment:
```suggestion
export enum ACTION_TYPE {
```

It should be singular, it represents a single value of multiple variants